crates/solidity-v2/outputs/cargo/common/src/lib.rs

```diff
@@ -0,0 +1 @@
+pub mod language_versions;
```

Comment:
I suggest renaming the module `language_versions/versions` to `versions/language_version`, as we will be migrating other version-related APIs to it over time (like the ones under the old `LanguageFacts`).